### PR TITLE
add a few more papers on sequential / flow-sensitive effect systems

### PR DESCRIPTION
**Pull request by csgordon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -875,6 +899,9 @@ in various programming languages.
   ([bibtex](http://dblp.uni-trier.de/rec/html/journals/corr/BauerP13?view=bibtex))
   ([doi](http://dx.doi.org/10.2168/LMCS-10%284:9%292014))
 
+* **Parametric Effect Monads and Semantics of Effect Systems** (POPL 2014)
+  by Shin-ya Katsumata
```
Comment (dhil) [inline README.md:903]:
```suggestion
* **Parametric Effect Monads and Semantics of Effect Systems** (POPL 2014)  
  by Shin-ya Katsumata  
```

File: README.md [hunk 2]
```diff
@@ -914,6 +941,10 @@ in various programming languages.
   ([doi](https://doi.org/10.1016/j.entcs.2013.09.007))
   ([pdf](https://danel.ahman.ee/papers/mfps13.pdf))
 
+* **The Sequential Semantics of Producer Effect Systems** (POPL 2013)  
+  by Ross Tate
```
Comment (dhil) [inline README.md:945]:
```suggestion
* **The Sequential Semantics of Producer Effect Systems** (POPL 2013)  
  by Ross Tate  
```

File: README.md [hunk 3]
```diff
@@ -988,6 +1019,10 @@ in various programming languages.
   by Wouter Swierstra  
   ([pdf](https://www.cambridge.org/core/services/aop-cambridge-core/content/view/14416CB20C4637164EA9F77097909409/S0956796808006758a.pdf))
 
+* **Types and trace effects of higher-order programs** (JFP 2008)
+  by Christian Skalka, Scott Smith, and David van Horn
```
Comment (dhil) [inline README.md:1023]:
```suggestion
* **Types and trace effects of higher-order programs** (JFP 2008)  
  by Christian Skalka, Scott Smith, and David van Horn  
```

**Review by dhil — COMMENTED** [review]
Thanks! Some entries are missing significant whitespace.

**Comment by csgordon** [comment]
Sorry I missed those, I've committed your fixes.

**Comment by dhil** [comment]
Thanks!

**Pull request merged**